find_symbols.py

```diff
@@ -0,0 +1,78 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script finds the libnative-lib.so and figures out which objcopy to use.
+"""
+import getopt;
+import glob
+import os
+import subprocess
+import sys
+
+platforms = {
+   'wavevr',
+   'oculusvr',
+   'googlevr',
+   'noapi',
+   'svr',
+}
+
+objcopyMap = {
+   'arm' : 'tools/taskcluster/symbols/bin/arm-linux-androideabi-objcopy',
+   'arm64' : 'tools/taskcluster/symbols/bin/aarch64-linux-android-objcopy',
+   'x86' : 'tools/taskcluster/symbols/bin/i686-linux-android-objcopy',
+}
+
+count = 0
+
+def find_platform(path):
+   global count
+   global platforms
+   values = path.split('/')
+   for part in values:
+      for platform in platforms:
+         if part.find(platform) == 0:
+            return part.lower()
+   count = count + 1
+   return 'unknown-platform-' + str(count)
```

Comment:
what is the `count` for? should we just give a sys.exit(1) here instead of continuing to run to the following functions?

Reply:
yep, I'll fix it. Originally I didn't realize we needed different objcopy binaries so if the platform wasn't know I figured it would still work if we added a new platform. But since we need to know the arch it should exit on unknown platform.